package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ux-pattern-library",
-  "version": "0.2.0",
+  "version": "0.1.46",
```

Comment:
Jigga what? Are we going back in time in version numbers for our own UX PL? Also, we should figure out when to set these.